bin/build

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -e
 
-go install github.com/Originate/git-town/cmd/gt
+go install github.com/Originate/git-town
```

Comment:
This script is always called from the root directory (otherwise it is not in the path). A simple `go install` is all that's necessary here.